src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+import java.util.Scanner;
+
 public class Duke {
+    // ========================================= GLOBAL CONSTANT
+    private static final int MAXIMUM_NUMBER_OF_TASKS = 100;
+
+    private static final String MESSAGE_EXIT = "Bye. Hope to see you again soon!";
+    private static final String MESSAGE_HELLO = "Hello! I'm a chatbot Duke made by Than Duc Huy\n" +
+            "Type the command to start interacting with Duke";
+    private static final String MESSAGE_COMMAND_LISTS = "Supported commands: list, mark, unmark, todo, deadline, event, bye";
+    private static final String MESSAGE_DIVIDER = "===================================================";
+    private static final String MESSAGE_DIVIDER_LIST = "=======================LIST========================";
+    
+    
+    private static final String MESSAGE_UNKNOWN_COMMAND = "Unknown Command";
+
+    private static final String ERROR_MESSAGE_BYE = "";
+    private static final String ERROR_MESSAGE_LIST = "";
+    private static final String ERROR_MESSAGE_MARK = "Syntax for mark \n\t>>> mark <item index number> \nNote: item index must exist in the current list";
+    private static final String ERROR_MESSAGE_UNMARK = "Syntax for unmark \n\t>>>unmark <item index number> \nNote: item index must exist in the current list";
+    private static final String ERROR_MESSAGE_TODO = "Syntax for todo \n\t>>>todo <task>";
+    private static final String ERROR_MESSAGE_DEADLINE = "Syntax for deadline \n\t>>>deadline <task> / <date of deadline>";
+    private static final String ERROR_MESSAGE_EVENT = "Syntax for event \n\t>>>event <task> / <date of event>";
+
+    // ========================================= GLOBAL VARIABLES
```

Comment:
The style of commenting may not be the best for CS2113. Can you consider the format given in the coding standard?